package.json

```diff
@@ -53,7 +53,6 @@
     "compile:trezor-dev": "TREZOR_DEV=true babel --extensions .ts,.js --watch -d lib/ src/ && npm run compile-images",
     "compile:watch": "babel --watch --extensions .ts,.js -d lib/ src/ && compile-images",
     "prepublish": "npm run compile",
-    "pretest": "npm run check-types && npm run compile",
```

Comment:
Typescript type checking is kind of like a test of sorts.  Add check-types to test script?

Reply:
Done  6035a7c
I think that was the original idea. Forget why I removed. `npm run compile` might've been messing stuff up.